cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -166,9 +174,13 @@ def test_retired_service(appliance, context):
     with appliance.context.use(context):
         appliance.server.login()
         dashboard = Dashboard(appliance)
-        assert dashboard.retired_services() == dashboard.num_of_rows()
+        if current_version() >= '5.8':
```

Comment:
@mfalesni this one is a interesting api difference - i wonder if there is a way to even that one out in a more general way

Reply:
@RonnyPfannschmidt just `appliance.version` ? As for the method rename, I don't know why it is done that way.

Reply:
@sshveta what is the difference between num_of_rows and results? Why is this change necessary? Can there be a wrapper method to decide? Can one of the methods be able to work under both versions?

Reply:
Also, `retired_services()` does not hint you it should be returning an integer. One would expect it to return a list of objects.